### PR TITLE
Korean chapter0, 1 full

### PR DESCRIPTION
**Pull request by Doohae**
Hello, I'm the assignee of Korean translation(chapter 1)
I just created _toctree.yml file and finished translation for ko/chapter1/1.mdx
Please check out for the translation and the form!
Planned to gotta finish all other part of chapter1 in a day 😄

Related to issue #42

### CODE REVIEW

File: chapters/ko/_toctree.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
+- title: 0. Setup
```
Comment (lewtun) [inline chapters/ko/_toctree.yml:1]: Feel free to translate this title if you want :)

File: chapters/ko/_toctree.yml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  sections:
+  - local: chapter0/1
+    title: Introduction
```
Comment (lewtun) [inline chapters/ko/_toctree.yml:4]: Same here

File: chapters/ko/_toctree.yml [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+- title: 1. 트랜스포머 모델
+  sections:
+  - local: chapter1/1
+    title: 단원 소개
+  - local: chapter1/2
```
Comment (lewtun) [inline chapters/ko/_toctree.yml:10]: We only need the sections of the table of contents for content that is already translated. In this case, you can remove everything from this section onwards :)

**Review by lewtun — COMMENTED** [review]
Thank you for this contribution @Doohae 🚀 ! I'll ask someone in the Korean community for a quick review, but in the meantime you can make some small tweaks to the table of contents - thank you!

File: chapters/ko/_toctree.yml (hunk 3 above)
Comment (Doohae) [reply chapters/ko/_toctree.yml:10]: I just fix all the things you mentioned above & completed chapter0 and 1!
Please check out!

File: chapters/ko/chapter1/4.mdx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+- **2018년 6월**: [GPT](https://cdn.openai.com/research-covers/language-unsupervised/language_understanding_paper.pdf), 최초로 사전 학습된 트랜스포머 모델로 다양한 NLP 작업에 미세 조정(fine-tune)되도록 사용되었으며 SOTA(state-of-the-art) 성능 달성
+- **2018년 10월**: [BERT](https://arxiv.org/abs/1810.04805), 또 다른 거대 사전 학습 언어 모델로, 더 좋은 문장 요약을 위해 설계 (이번 단원과 다음 단원에서 더 자세히 알아봐요!)
+- **2019년 2월**: [GPT-2](https://cdn.openai.com/better-language-models/language_models_are_unsupervised_multitask_learners.pdf), 더 좋은 성능(그리고 더 큰) 버전의 GPT로, 윤리적 문제로 인해 즉시 공개되지 못하였음
+- **2019년 10월**: [DistilBERT](https://arxiv.org/abs/1910.01108), BERT의 60% 빠른 속도에 메모리 측면에서 40% 가볍지만 BERT 성능의 97%를 재현하는 경량화 버전의 BERT
+- **2019년 10월**: [BART](https://arxiv.org/abs/1910.13461) 와 [T5](https://arxiv.org/abs/1910.10683), 동일한 구조의 (처음으로) 원본 트랜스포머 모델의 구조를 그대로 따른 두 거대 사전학습 언어 모델
+- **2020년 5월**, [GPT-3](https://arxiv.org/abs/2005.14165), 미세 조정 없이도 (zero-shot learning이라 부름) 다양한 작업을 훌륭하게 수행하는 GPT-2의 더 큰 버전
+
+This list is far from comprehensive, and is just meant to highlight a few of the different kinds of Transformer models. Broadly, they can be grouped into three categories:
```
Comment (rainmaker712) [inline chapters/ko/chapter1/4.mdx:23]: duplicated message in Eng.

**Review by rainmaker712 — CHANGES_REQUESTED** [review]
There are few parts need to be changed + double check for url & image links.

File: chapters/ko/chapter0/1.mdx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+설치에 시간이 조금 걸리겠지만 이제 나머지 강의를 위한 준비를 모두 마쳤습니다!
+
+## 파이썬 가상 환경 사용하기
+
+파이썬 가상 환경 사용을 원하신다면 우선 시스템에 파이썬을 설치해야 합니다. 이 [가이드](https://realpython.com/installing-python/)를 따라 시작해보세요.
+
+파이썬 설치가 완료되면 터미널에서 파이썬 명령어를 실행할 수 있습니다. 다음 단계로 넘어가기 전에, 다음과 같은 명령어를 실행하여 설치가 잘 되었는지 확인하세요: `python --version`. 이 때 시스템에 사용할 수 있는 파이썬 버전을 출력되어야 합니다.
+
+터미널에서 `python --version` 과 같은 파이썬 명령어를 실행하면, 명령어를 실행하는 프로그램을 시스템의 “메인(main)” 파이썬으로 생각해야 합니다. 이 메인 설치에 패키지를 설치하지 않은 채로 놔두시고, 이를 작업 중인 각 어플리케이션에 별도의 환경을 생성하는 데에 이용하는 것을 추천드립니다. 이렇게 하면, 각 어플리케이션은 각각의 의존성과 패키지를 갖게 되어 서로 다른 어플리케이션과 추후에 호환성 문제로부터 자유롭게 됩니다.
```
Comment (bzantium) [inline chapters/ko/chapter0/1.mdx:58]:
```suggestion
터미널에서 `python --version` 과 같은 파이썬 명령어를 실행하면, 명령어를 실행하는 프로그램을 시스템의 “메인(main)” 파이썬으로 생각해야 합니다. 이 메인 파이썬은 어떤 패키지도 설치하지 않은 상태로 유지하면서, 작업 중인 각 어플리케이션마다 별도의 환경을 생성하여 이용하는 것을 권장합니다. 이렇게 하면, 각 어플리케이션은 각각의 의존성 및 패키지를 갖게 되어 다른 어플리케이션과의 잠재적 호환성 문제를 피할 수 있습니다.
```

File: chapters/ko/chapter0/1.mdx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+파이썬에서 이는 *[가상 환경](https://docs.python.org/3/tutorial/venv.html)*을 통해 완수됩니다. 가상 환경은 자체 포함 디렉토리 트리로, 각 트리는 어플리케이션에게 필요한 모든 패키지와 함께 특정 파이썬 버전에 대한 파이썬 설치를 포함합니다. 이러한 가상 환경을 생성하는 방법은 여러 툴을 통해 할 수 있지만, 여기서는 공식 파이썬 패키지인 `[venv](https://docs.python.org/3/library/venv.html#module-venv)` 를 통해 생성해 보겠습니다.
+
+먼저, 여러분의 어플리케이션을 넣어줄 디렉토리를 생성하세요. 예를 들어, 여러분의 홈 디렉토리의 *transformers-course*와 같은 이름의 디렉토리를 만들어 볼까요?:
```
Comment (bzantium) [inline chapters/ko/chapter0/1.mdx:62]:
```suggestion
먼저, 어플리케이션을 넣어줄 디렉토리를 생성합니다. 예를 들어, 홈 디렉토리의 *transformers-course*와 같은 이름의 디렉토리를 만들어 봅시다:
```

File: chapters/ko/chapter0/1.mdx [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+```
+# Activate the virtual environment
+source .env/bin/activate
+
+# Deactivate the virtual environment
+source .env/bin/deactivate
+
+```
+
+환경이 제대로 활성화 되는지 `which python` 명령어를 실행하여 확인해보세요. 아래와 같이 가상 환경을 가리킨다면 활성화가 완료된 것입니다!
```
Comment (bzantium) [inline chapters/ko/chapter0/1.mdx:100]:
```suggestion
환경이 제대로 활성화 되었는지 `which python` 명령어를 실행하여 확인해 봅시다. 아래와 같이 가상 환경을 보여준다면 제대로 활성화가 것입니다!
```

File: chapters/ko/chapter0/1.mdx [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+### 의존성(dependencies) 설치하기
+
+Google Colab 사용법에서와 마찬가지로 다음 단계로 넘어가기 위해 패키지를 설치해야 합니다. 여기서도, `pip` 패키지 관리자를 통해  🤗 Transformers 개발 버전을 설치할 수 있습니다:
+
+```
+pip install "transformers[sentencepiece]"
+
+```
+
+이제 모든 설정이 끝났고 시작할 준비를 마쳤습니다!
```
Comment (bzantium) [inline chapters/ko/chapter0/1.mdx:121]:
```suggestion
이제 모든 환경 설정믈 마치고 시작할 준비가 되었습니다!
```

**Review by bzantium — CHANGES_REQUESTED** [review]
Review chapter0
- align all expressions: using "합니다" instead of "요"
- revise english-like expressions (일부 의역)

File: chapters/ko/chapter0/1.mdx [hunk 9]
```diff
@@ -0,0 +1,121 @@
+# 강의 소개
+
+Hugging Face 강의에 오신 여러분들 환영합니다! 이번 강의 소개에서는 작업 환경 설정에 대해 안내드리겠습니다. 방금 막 이번 과정을 시작하셨다면 먼저 [Chapter 1](notion://www.notion.so/course/chapter1) 내용을 살펴보고 돌아오신 뒤, 환경을 설정하여 코드를 직접 실행해보시길 추천드립니다.
```
Comment (lewtun) [inline chapters/ko/chapter0/1.mdx:3]: Does this Notion link exist? I think the following should work:

```suggestion
Hugging Face 강의에 오신 여러분들 환영합니다! 이번 강의 소개에서는 작업 환경 설정에 대해 안내드리겠습니다. 방금 막 이번 과정을 시작하셨다면 먼저 [Chapter 1](/course/ko/chapter1) 내용을 살펴보고 돌아오신 뒤, 환경을 설정하여 코드를 직접 실행해보시길 추천드립니다.
```

**Review by lewtun — COMMENTED** [review]
Thanks for integrating the PR feedback @Doohae ! I noticed one of the URLs is pointing to the wrong place, but otherwise this looks good 🤗 

Once we've fixed the URL, I think we can merge this!

File: chapters/ko/chapter1/10.mdx [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+
+그래도 우선, 이번 단원에서 배운 내용에 대해 확인해보는 시간을 갖도록 하겠습니다!
+
+### 1. Hub에서 `roberta-large-mnli` 체크포인트를 검색해 보세요. 이 모델은 어떤 작업을 수행하나요?
+
+<Question
+choices={[
+{
+text: "요약",
+explain: "이 페이지를 다시 확인하세요 <a href=\"[https://huggingface.co/roberta-large-mnli\\](https://huggingface.co/roberta-large-mnli%5C%5C)">roberta-large-mnli page</a>."
```
Comment (lewtun) [inline chapters/ko/chapter1/10.mdx:15]:
```suggestion
explain: "이 페이지를 다시 확인하세요 <a href=\"https://huggingface.co/roberta-large-mnli\">roberta-large-mnli page</a>."
```

File: chapters/ko/chapter1/10.mdx [hunk 11]
```diff
@@ -0,0 +1,253 @@
...
+},
+{
+text: "텍스트 분류",
+explain: "더 정확하게 말하면, 이 모델은 두 문장이 논리적으로 타당한지 세 가지 레이블(모순, 함의, 중립)로 분류합니다. 이러한 문제를 <em>자연어 추론(natural language inference)</em>이라고 부릅니다.",
+correct: true
+},
+{
+text: "텍스트 생성",
+explain: "이 페이지를 다시 확인하세요 <a href=\"[https://huggingface.co/roberta-large-mnli\\](https://huggingface.co/roberta-large-mnli%5C%5C)">roberta-large-mnli page</a>."
```
Comment (lewtun) [inline chapters/ko/chapter1/10.mdx:24]: We need to change this here to build the docs properly :)

```suggestion
explain: "이 페이지를 다시 확인하세요 <a href=\"https://huggingface.co/roberta-large-mnli\">roberta-large-mnli page</a>."
```

File: .github/workflows/build_documentation.yml [hunk 12]
```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es fr he pt th tr
+      languages: en es fr he ko pt th tr
```
Comment (lewtun) [inline .github/workflows/build_documentation.yml:17]: FYI I've added Korean to the list here so we can build the docs on the website once this PR is merged :)

**Comment by lewtun** [comment]
Hey @Doohae we have some merge conflicts in the GitHub CI files due to some recent changes from other translations. Would you mind resolving them so we can try building the docs on this PR? Thanks!

**Comment by Doohae** [comment]
> Hey @Doohae we have some merge conflicts in the GitHub CI files due to some recent changes from other translations. Would you mind resolving them so we can try building the docs on this PR? Thanks!

Hey I just resolved them and also fixed the url&tag typos, indentation, and unnatural expressions!
Really sorry for being late on the revision 😭

**Review by lewtun — APPROVED** [review]
This is awesome! So exciting to see the first part of the course in Korean :)

If the tests pass in the CI, I'll deploy this later today!

**Comment by lewtun** [comment]
The docs render beautifully, so merging this!

Screen Shot 2022-04-06 at 16 49 15

**Pull request merged**